src/lib/y2packager/clients/inst_productsources.rb

```diff
@@ -412,10 +401,9 @@ def InitializeSources
       Pkg.TargetInitialize(Installation.destdir)
       # the fastest way
       Pkg.SourceRestore
-
       if !Mode.installation
         # repos_already_used
-        Builtins.foreach(Pkg.SourceGetCurrent(true)) do |one_id|
+        Builtins.foreach(Pkg.SourceGetCurrent(false)) do |one_id|
```

Comment:
OK, so also check the disabled repositories...

Reply:
Yes, I think that should be OK.